site/content/docs/include/sidecar-config.en.md

```diff
@@ -2,8 +2,41 @@
 <a id="port" href="#port" class="field">`port`</a> <span class="type">Integer</span>  
 Port of the container to expose (optional).
 
-<a id="image" href="#image" class="field">`image`</a> <span class="type">String</span>  
-Image URL for the sidecar container (required).
+<a id="image" href="#image" class="field">`image`</a> <span class="type">String or Map</span>  
```

Comment:
Maybe we can reuse the one for main container if our implementation uses the same struct 🤔

Reply:
just a small difference these two is `image` field  is `string` for main container and for sidecar it is `String or Map`

Reply:
Done. used the same docs of main container.